src/App.js

```diff
@@ -1,8 +1,26 @@
 import './App.css';
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Home from './components/home';
+import Quote from './components/quote';
+import Navbar from './components/Navbar';
 
 function App() {
-  return (<Calculator />);
+  return (
+    <Router>
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/calculator" element={<Calculator />} />
```

Comment:
- [ ] Following the project requirements, the general layout should match the given wireframes in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md). On the calculator page, there should be text on the left and the calculator on the right.